test/smoke-test.js

```diff
@@ -35,13 +36,72 @@ describe('smoke test', () => {
   it('should get a details report', async () => {
     const client = togglClient();
 
+    // FIXME: Add a time entry before to build a fixture
     const workspaces = await client.workspaces.list();
+    const detailsReport = await client.reports.details(workspaces[0].id, {
+      start_date: dayjs().subtract(1, 'week').format('YYYY-MM-DD'),
+    });
+
+    debug(detailsReport[0]);
+    expect(detailsReport).to.be.an('array');
+    expect(detailsReport[0]).to.have.property('user_id');
+    expect(detailsReport[0]).to.have.property('username');
+    expect(detailsReport[0]).to.have.property('project_id');
+    expect(detailsReport[0]).to.have.property('task_id');
+    expect(detailsReport[0]).to.have.property('description');
+    expect(detailsReport[0]).to.have.property('time_entries');
+    expect(detailsReport[0]).to.have.property('row_number');
+  });
 
-    const detailsReport = await client.reports.details(workspaces[0].id);
+  it('should throw an error if a start date is not provided with a details report', async () => {
+    const client = togglClient();
 
-    debug(detailsReport);
+    const workspaces = await client.workspaces.list();
 
-    expect(detailsReport).to.exist.to.be.an('object');
+    try {
```

Comment:
@beauraines fixed the tests by using good old plain try/catch blocks and failing in case no exception is thrown